### ISSUE TITLE
Rollup integration uses `name`, `path` instead of `find`, `replacement`

### ISSUE THREAD

**Issue by IanVS** [davestewart/alias-hq#32]
This seems like a very cool tool, and I'm getting started using it, but having some trouble with rollup.

I'm building my app with vite, which allows a `resolve.alias` setting (https://vitejs.dev/config/#resolve-alias) that is passed to @rollup/plugin-alias.  However, when I run `hq.get('rollup')`, I get an array like this:

```
[
  { name: '@api', path: '/path/to/src/api' },
  {
    name: '@components',
    path: '/path/to/src/components'
  }
]
```

But rollup wants the objects to have keys of `find` and `replacement`.  Debugging via the cli results in the right thing, just not `hq.get()`. Am I doing something incorrectly?  Thanks!

Edit:  I just found that `hq.get('rollup', { format: 'object' })` will meet my needs, but I'll leave this open in case you'd like to look into it.

**Comment by davestewart** [comment]
Hey Ian,

Thanks for letting me know!

Actually, I've just come back to this project after about a year and it wasn't easy to find that information.

I outline all the options here:

- https://github.com/davestewart/alias-hq/blob/master/docs/integrations.md

I'm not sure why, but it's not easy to find if you just come to GitHub!

I'll see what I can do about that now.

Cheers,
Dave

**Comment by IanVS** [comment]
Thanks, I see two issues:

1) the array format produces incorrect keys for rollup
2) the docs indicate that object is the default format, but it is not, array is the default.

I'd be happy to submit a PR if you like.

**Comment by davestewart** [comment]
> the array format produces incorrect keys for rollup

I don't think that's the case.

I just ran the "Debug integrations" CLI option, and this is what it outputs:

```
  -- Integrations Menu --

? What do you want to do? Debug integration
? Which integration? Rollup
? Which format? Object

    {
      "@": "/Volumes/Data/Work/OpenSource/JavaScript/AliasHQ/alias-hq/demo/src",
      "@packages": "/Volumes/Data/Work/OpenSource/JavaScript/AliasHQ/alias-hq/demo/packages",
      "@classes": "/Volumes/Data/Work/OpenSource/JavaScript/AliasHQ/alias-hq/demo/src/classes",
      "@app": "/Volumes/Data/Work/OpenSource/JavaScript/AliasHQ/alias-hq/demo/src/app",
      "@data": "/Volumes/Data/Work/OpenSource/JavaScript/AliasHQ/alias-hq/demo/src/app/data",
      "@services": "/Volumes/Data/Work/OpenSource/JavaScript/AliasHQ/alias-hq/demo/src/app/services",
      "@views": "/Volumes/Data/Work/OpenSource/JavaScript/AliasHQ/alias-hq/demo/src/app/views"
    }
    

  -- Integrations Menu --

? What do you want to do? Debug integration
? Which integration? Rollup
? Which format? Array

    [
      {
        "find": "@",
        "replacement": "/Volumes/Data/Work/OpenSource/JavaScript/AliasHQ/alias-hq/demo/src"
      },
      {
        "find": "@packages",
        "replacement": "/Volumes/Data/Work/OpenSource/JavaScript/AliasHQ/alias-hq/demo/packages"
      },
      {
        "find": "@classes",
        "replacement": "/Volumes/Data/Work/OpenSource/JavaScript/AliasHQ/alias-hq/demo/src/classes"
      },
      {
        "find": "@app",
        "replacement": "/Volumes/Data/Work/OpenSource/JavaScript/AliasHQ/alias-hq/demo/src/app"
      },
      {
        "find": "@data",
        "replacement": "/Volumes/Data/Work/OpenSource/JavaScript/AliasHQ/alias-hq/demo/src/app/data"
      },
      {
        "find": "@services",
        "replacement": "/Volumes/Data/Work/OpenSource/JavaScript/AliasHQ/alias-hq/demo/src/app/services"
      },
      {
        "find": "@views",
        "replacement": "/Volumes/Data/Work/OpenSource/JavaScript/AliasHQ/alias-hq/demo/src/app/views"
      }
    ]
```

Using the helper function, you can pass that into your rollup config by itself, or mix it in with additional config:

```js
// rollup.config.js
import hq from 'alias-hq'
import alias from '@rollup/plugin-alias';

module.exports = {
  ...
  plugins: [
    alias({
      entries: {
        ...hq.get('rollup') // object is default, but you can use array by passing params
      }
    })
  ]
}
```


> the docs indicate that object is the default format, but it is not, array is the default.

I'm not sure what you mean here.

The docs [give two options](https://github.com/rollup/plugins/tree/master/packages/alias#options), both work.

One has to be "the default" and I went with what Rollup put first (and what is the briefest) on their own docs.


Unless I'm missing what you're trying to tell me...

**Comment by davestewart** [comment]
I've just spent some time looking into this, didn't quite understand what you're alluding to, but I think I finally got there.

The problem is here:

https://github.com/davestewart/alias-hq/blob/master/src/plugins/rollup/index.js#L9

The merging of defaults should actually be done here:

https://github.com/davestewart/alias-hq/blob/master/src/plugins/rollup/index.js#L29

Otherwise, if you skip passing options, the main function returns the array format – as you said.

Thanks for flagging! I'll get this fixed up and published in the next hour or so.

**Comment by davestewart** [comment]
OK, looking good:

![image](https://user-images.githubusercontent.com/132681/139543238-a196ec12-a5f5-4b80-94e0-5ff06c6d30f3.png)

About to publish.

**Comment by davestewart** [comment]
OK! Fixes published as 5.1.7:

https://www.npmjs.com/package/alias-hq

Thanks for the flagging.
